adapters/jdbc/src/test/java/org/atomhopper/jdbc/adapter/JdbcFeedSourceTest.java

```diff
@@ -137,7 +137,7 @@ public void shouldGetCurrentLinkFromArchiveFeedAndArchiveNode() throws Exception
             when(getFeedRequest.getDirection()).thenReturn("forward");
             when(getEntryRequest.getAbdera()).thenReturn(localAbdera);
             when(jdbcTemplate.query(any(String.class), any(Object[].class), any(EntryRowMapper.class))).thenReturn(entryList);
-            when(jdbcTemplate.queryForInt(any(String.class), any(Object[].class))).thenReturn(1);
+            when(jdbcTemplate.queryForObject(eq(any(String.class)), any(Object[].class), Integer.class)).thenReturn(1);
```

Comment:
It didn't make any sense to me that these only needed to be changed in the mocks but not in any implementation code, so I commented out every single line in the class that included `queryFor` to see what happens. The tests continued to pass, so we should probably just remove these lines.

I didn't check the other test class, but I bet it's worth verifying if it's not needed there, too.